core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/ColumnPageDecoder.java

```diff
@@ -29,6 +31,12 @@
    */
   ColumnPage decode(byte[] input, int offset, int length) throws MemoryException, IOException;
 
+  /**
+   *  Apply decoding algorithm on input byte array and fill the vector here.
+   */
+  void decodeAndFillVector(byte[] input, int offset, int length, ColumnVectorInfo vectorInfo,
+      BitSet nullBits, boolean isLVEncoded) throws MemoryException, IOException;
```

Comment:
I feel it is not good to add `isLVEncoded` just for LVEncoded, can we pass a more generic parameter, since this is a common class for all Decoder

Reply:
Yes,  it is as per the old method `decode` added this method, It was added as part of the local dictionary and it is getting refactored as part of vishal's store method.
@kumarvishal09 please remove this `isLVEncoded` from decode method. And it should align one of the datatypes used in the column page in case of local dictionary.